codespell_lib/data/dictionary.txt

```diff
@@ -57035,6 +57035,11 @@ trasposed->transposed
 trasposing->transposing
 trasposition->transposition
 traspositions->transpositions
+trasversal->traversal, transversal,
```

Comment:
Perhaps add plural of nouns:
```suggestion
trasversal->traversal, transversal,
trasversals->traversals, transversals
```

Reply:
Yes, absolutely. As the code suggestion above didn't included the trailing `,` i have just force pushed a new variant of this PR including the suggestion.

Thanks a lot for it :+1: